src/sentry/conf/server.py

```diff
@@ -1343,3 +1344,11 @@ def get_raven_config():
 # Relay
 # List of PKs whitelisted by Sentry
 SENTRY_RELAY_WHITELIST_PK = []
+
+# CDN
+# If this is an absolute url like e.g.: https://js.sentry-cdn.com/
+# the full url will look like this: https://js.sentry-cdn.com/<public_key>.min.js
+# otherwise django reverse url lookup will be used.
+JS_SDK_LOADER_CDN_URL = ''
+JS_SDK_LOADER_SDK_VERSION = '0.5.2'
+JS_SDK_LOADER_DEFAULT_SDK_URL = 'https://s3.amazonaws.com/getsentry-cdn/@sentry/browser/%s/bundle.min.js' % JS_SDK_LOADER_SDK_VERSION
```

Comment:
I assume this is just a placeholder? We're not serving this directly from S3.

Reply:
Ehm, yup this is basically a placeholder, it is a working bundle but in no way final. The loader as it works now can only handle the new JS SDK, so this will be changed.